domains/krvishal.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Portfolio",
+    "repo": "https://github.com/thegeekdex",
+    "owner": {
+        "username": "thegeekdex",
+        "email": "",
+        "discord": "Vishal S.#6993"
```

Comment:
This is optional but recommended:

It is best to include your Discord ID, just incase your tag changes.
Like this: `"note": "My Discord ID is: 1234567890"`